Code/Framework/AzToolsFramework/AzToolsFramework/UI/Prefab/PrefabUiHandler.cpp

```diff
@@ -417,24 +444,20 @@ namespace AzToolsFramework
     bool PrefabUiHandler::OnOutlinerItemClick(const QPoint& position, const QStyleOptionViewItem& option, const QModelIndex& index) const
     {
         AZ::EntityId entityId = GetEntityIdFromIndex(index);
-        const QPoint offset = QPoint(-18, 3);
+        const QPoint textOffset = QPoint(0, 3);
 
-        if (m_prefabFocusPublicInterface->IsOwningPrefabInFocusHierarchy(entityId))
+        QRect filenameRect = QRect(0, 0, 12, 10);
+        filenameRect.translate(option.rect.topRight() + QPoint(-13, 7) + textOffset);
```

Comment:
Can we document these hard-coded numbers?

Reply:
Sorry I really don't know how @AMZN-daimini decided these numbers.